ragna/deploy/_cli/core.py

```diff
@@ -171,3 +177,215 @@ def wait_for_api() -> None:
         if process is not None:
             process.kill()
             process.communicate()
+
+
+class SourceStorageImport(BaseModel):
+    source_storage_class: ImportString
+
+
+class DocumentFilesystemHandlerImport(BaseModel):
+    document_filesystem_handler_class: ImportString
+
+
+@corpus_app.command(help=" Ingest some documents into a given corpus.")
+def ingest(
+    source_storage: Annotated[
+        str,  # typer does not support ImportString
+        typer.Option(
+            "--source-storage",
+            metavar="SOURCE_STORAGE",
+            default_factory=lambda: "ragna.source_storages.RagnaDemoSourceStorage",
+            help="Select source for storing vector embeddings.",
+        ),
+    ],
+    document_filesystem_handler: Annotated[
+        str,  # typer does not support ImportString
+        typer.Option(
+            "--document-filesystem-handler",
+            metavar="DOCUMENT_FILESYSTEM_HANDLER",
+            default_factory=lambda: "ragna.core.LocalDocument",
```

Comment:
From #488: default to ingest files from local disk unless user supplies a different document filesystem handler.